docs/builds.rst

```diff
@@ -14,6 +14,9 @@ Our current build limits are:
 
 We can increase build limits on a per-project basis,
 sending an email to support@readthedocs.org providing a good reason why your documentation needs more resources.
+If your business is hitting build limits hosting documentation on Read the Docs,
```

Comment:
This is tricky to me. I mean, "if you have a _business_, you should not be using .org" I understand.

Although, maybe _business_ has a more expand meaning and I'm not getting it. I immediately related it to _company_, so.

Reply:
Well, I think the bigger issue here is businesses asking for free support (additional resources, specific issues in the issue tracker, etc.) for running their critical business processes. If they want that support, there's a way to get it: paid plans. For a business that is just using Read the Docs and doesn't really need anything, have at it!